### PR TITLE
Feature/weekly reset

### PR DESCRIPTION
**Pull request by ajg7**
# Description

Fixes #

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by gregory-j-wilson — CHANGES_REQUESTED** [review]
is all the stuff deleted ok to delete?

**Review by ajg7 — COMMENTED** [review]
Yep that stuff is ok to delete

**Comment by gregory-j-wilson** [comment]
great.

**Review by gregory-j-wilson — DISMISSED** [review]

File: api/cronTasks/scheduler.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
 const cron = require('node-cron');
```
Comment (DeeDowns) [inline api/cronTasks/scheduler.js:1]: So what this is doing on schedule determining who the winning team is and then assigning the points to the child? So the total points won in the matchup only go to a child of the winning team? Or the points go to the child who's submission was voted best?

**Review by DeeDowns — DISMISSED** [review]

**Review by Nicci498 — DISMISSED** [review]

**Review by gregory-j-wilson — APPROVED** [review]
cool

**Review by DeeDowns — APPROVED** [review]

**Pull request merged**